### PR TITLE
#54 Implements SignUp Email

### PR DESCRIPTION
**Pull request by AuraOfDivinity**
**Problem**
Github Issue Number: #54 
Describe the problem you are trying to achieve. 
Sending and Email on successful SignUp

### **In Case of UI Changes**
**Before Changes Screenshot** [How was it earlier, add it here]

**After Changes Screenshot**

### **Type of Change**
- [ ] Bug fix
- [x] New Feature
- [ ] Development of UI/UX prototypes
- [ ] Small refactor
- [ ] Change in Documentation

### **Checklist**
- [x] My code follows the same style as the codebase
- [x] My Code change requires a change in documentation
- [x] I have updated the Readme accordingly
- [x] I made PR against **development branch**
- [x] I have run the test cases locally and it's passing.
- [x] I have squashed my commits

### CODE REVIEW

File: app/controllers/user.js [hunk 1]
```diff
@@ -1,11 +1,13 @@
 const User = require('../models/User')
 const jwt = require('jsonwebtoken')
+const emailServices = require('../services/email')
 
 module.exports = {
   createUser: async (req, res, next) => {
     const user = new User(req.body)
     try {
       await user.save()
+      emailServices.sendSuccessfulSignupEmail(req.body.email, req.body.name)
```
Comment (Rupeshiya) [inline app/controllers/user.js]: Please use the updated backend code. Check the user schema once, the name is an Object

Comment (Rupeshiya) [reply app/controllers/user.js]: That's why CI is failing.

Comment (AuraOfDivinity) [reply app/controllers/user.js]: Thanks @Rupeshiya. Didn't realize it was a object. Maybe we should change the content at the documentation as well as it's misleading. Already asked @devesh-verma about the key.

**Comment by Rupeshiya** [comment]
@AuraOfDivinity Also please add the activate acc button in email template

File: app/routes/user.js [hunk 2]
```diff
@@ -2,43 +2,23 @@ const express = require('express')
 const router = express.Router()
 const userController = require('../controllers/user')
 const auth = require('../middleware/auth')
+const email = require('../middleware/email')
```
Comment (devesh-verma) [inline app/routes/user.js:5]: can we revert the style change for this file.

File: app/controllers/user.js [hunk 3]
```diff
@@ -1,4 +1,4 @@
-const User = require('../models/User')
+const User = require('../models/User.js')
```
Comment (devesh-verma) [inline app/controllers/user.js:1]: remove .js

**Pull request merged**